app/auth/auth_service.ts

```diff
@@ -198,9 +198,8 @@ export class AuthService {
 
     this.setCookie(SELECTED_GROUP_ID_COOKIE, groupId);
 
-    // If we're on a subdomain and the new group is on a different subdomain then
-    // we have to use a redirect.
-    if (capabilities.config.customerSubdomain && new URL(groupURL).hostname != window.location.hostname) {
+    // If the new group is on a different subdomain then we have to use a redirect.
+    if (capabilities.config.subdomainsEnabled && new URL(groupURL).hostname != window.location.hostname) {
```

Comment:
quick sanity check, can the groupURL be empty here? (We have groups without url_identifier set, but not sure if groupURL would just be `app.buildbuddy.io` in that case)

Reply:
The backend sets the URL to `app.buildbuddy.io` if the group doesn't have an identifier.